### ISSUE TITLE
Do not link to pipeline if down

### ISSUE THREAD

**Issue by martinmosegaard** [Praqma/git-phlow#219]
The readme links to https://concourse.kubernetes.praqma.cloud/teams/main/pipelines/git-phlow
which has two issues:

- It is unsafe
- It is down

**Comment by groenborg** [comment]
@martinmosegaard Yes this is an error.. That is the old pipeline, which is not running anymore!

**Comment by buep** [comment]
simple fix - update docs at some point

**Comment by groenborg** [comment]
yes. I can set aside 10 minutes today and create a new milestone with cleanup tasks regarding our readme files and such @buep
